src/modules/users/edit-user/edit-user.html

```diff
@@ -100,8 +94,8 @@
                     </div>
                     <div class="radio radio-class">
                         <label>
-                            <input type="radio" name="role" id="role3" value="limited" ng-model="editUserCntrl.user.role"> <strong>Limited User</strong>
-                            <br> The Limited user role permits the user to access Tendrl in read-only mode, which means the user cannot actively manage storage. This role is ideal for users interested in monitoring the storage.
+                            <input type="radio" name="role" id="role3" value="limited" ng-model="editUserCntrl.user.role"> <strong>Read-only User</strong>
+                            <br> The Read-only user role permits the user to access Tendrl in read-only mode, which means the user cannot actively manage storage. This role is ideal for users interested in monitoring the storage.
```

Comment:
@a2batic As user is not allowed to update role. Please remove the html for it.
.